circup/__init__.py

```diff
@@ -1317,8 +1334,16 @@ def install(ctx, modules, pyext, requirement, auto, auto_file):  # pragma: no co
         with open(requirement, "r", encoding="utf-8") as rfile:
             requirements_txt = rfile.read()
         requested_installs = libraries_from_requirements(requirements_txt)
-    elif auto:
-        auto_file = os.path.join(ctx.obj["DEVICE_PATH"], auto_file)
+    elif auto or auto_file:
+        if auto_file is None:
+            auto_file = "code.py"
+        # pass a local file with "./" or "../"
+        is_relative = auto_file[:2] in ("." + os.sep, "..")
```

Comment:
The file could be `...foo`. How about:
```suggestion
        is_relative = auto_file.startswith("." + os.sep) or auto_file.startswith(".." + os.sep)
```

Reply:
Maybe this could be done with `any((auto_file.startswith(prefix + os.sep) for prefix in (".", "..)))` ?